index.html

```diff
@@ -9,20 +9,24 @@
 
   <script src="node_modules/core-js/client/shim.min.js"></script>
   <script src="node_modules/zone.js/dist/zone.js"></script>
+  <!--
```

Comment:
Got to remove a lot of js includes that aren't necessary once you're building a single bundle (you don't need any systemjs stuff)